packages/sdk-v2/src/sdk/goodid/hooks/useCertificates.ts

```diff
@@ -1,28 +1,71 @@
-import { useCallback } from "react";
-import { isEmpty } from "lodash";
-import { CredentialTypes } from "../types";
+import { useCallback, useContext, useEffect, useState } from "react";
+import { first } from "lodash";
 
 import type { Certificate } from "../types";
 
-// This expects a single issuer (gd) for the mvp purpose
-export const useCertificates = (db: any) => {
-  const getCertificates: (credentialTypes?: string[]) => Promise<Certificate[] | undefined> = useCallback(
-    async credentialTypes => {
-      const keys = credentialTypes ?? Object.keys(CredentialTypes);
-      return await db.certificates.where("type").anyOf(keys).toArray();
+import { GoodIdContext } from "../../../contexts";
+
+const cleanupCertificate = (value: any) => {
```

Comment:
@johnsmith-gooddollar changed because of typing issues

Reply:
revert, I want it as is. you could just do `{ primary_idx, type_subject_idx, ...certificate }: any` and add `as` cast to the end